src/extension.ts

```diff
@@ -357,6 +366,43 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
     configuration.initFromState();
     await configuration.initFromSettings(true);
 
+    context.subscriptions.push(vscode.commands.registerCommand('makefile.outline.openMakefilePathSetting', () => {
+        return vscode.commands.executeCommand("workbench.action.openSettings", "makefile.makefilePath");
```

Comment:
I suggest "openWorkspaceSettings" instead of "openSettings". For all occurrences.

Reply:
Done! 👍